integration-test/Css.test.tsx

```diff
@@ -221,6 +221,52 @@ describe("Css", () => {
       }
     `);
   });
+
+  it("can use generated breakpoints with emotion", () => {
+    const r = render(<div css={{ ...Css.mb1.pb2.$, [sm]: Css.pb3.$ }} />);
+    expect(r.container).toMatchInlineSnapshot(`
+      .emotion-0 {
+        margin-bottom: 8px;
+        padding-bottom: 16px;
+      }
+
+      @media screen and (max-width:599px) {
+        .emotion-0 {
+          padding-bottom: 24px;
+        }
+      }
+
+      <div>
+        <div
+          class="emotion-0"
+        />
+      </div>
+    `);
+  });
+
+  it("can use generated breakpoints with if dsl", () => {
+    expect(Css.pb2.white.if(sm).pb3.black.$).toMatchInlineSnapshot(`
+      Object {
+        "@media screen and (max-width:599px)": Object {
+          "color": "#353535",
+          "paddingBottom": "24px",
+        },
+        "color": "#fcfcfa",
+        "paddingBottom": "16px",
+      }
+    `);
+  });
+
+  it("will not accept just random strings to if", () => {
+    // @ts-expect-error
+    Css.black.if("rawstring").$;
+  });
+
+  it("cannot 'else' when using `if(bp)`", () => {
+    expect(() => Css.if(sm).black.else.white.$).toThrow(
+      "else is not supported"
+    );
+  });
```

Comment:
Makes sense. Maybe the error response should be something like "Cannot use `else` on breakpoint conditionals. Apply these styles before the conditional to achieve the expected behavior."

Reply:
Makes sense.

Fwiw I've since had the thought that, I bet at codegen time I could actually know the "inverse" of each breakpoint abbreviation (i.e. `mdAndDown` inverse --> `lgAndUp`), and so go back to supporting `else` by having a "breakpoint abbreviation --> the opposite version of that abbreviation" mapping.

That'd work for truss's codegen media queries, but if the `if` here started supporting random media queries, we wouldn't know how to "else" those. Granted, could handle that when we get there.